foundation/auth/Cargo.toml

```diff
@@ -36,7 +36,9 @@ hex = { version = "0.4", optional = true }
 tokio = { version = "1.32", features = ["test-util", "rt-multi-thread", "macros"]}
 tracing-subscriber = {version="0.3", features=["env-filter","std"]}
 ctor = "0.1"
-serial_test = "0.9"
+tempdir = "0.3.7"
```

Comment:
Cloud you remove tempdir?
tempdir is archived and has volunerability.
https://rustsec.org/advisories/RUSTSEC-2018-0017.html

Reply:
Ah yes, good catch, thanks :)